static/customWidgets/RemoteAutocomplete/remoteautocomplete.js

```diff
@@ -30,7 +30,7 @@ $(function () {
                 x: f.attrs.x,
                 y: f.attrs.y,
                 id: f.attrs.featureId,
-                label: f.attrs.label.replace("<b>", " - ").replace("</b>", ""),
+                label: f.attrs.label.replace(/\s\<b\>\s*/g, ", ").replace(/\s*[\<\(][^\)\>]*[\)\>]*/g, ""),
```

Comment:
This replace is too large (try to search a commune with `()`, it removes it, like `Granges (Veveyse)`).

Try something to remove explicitly the EGRID number (like `<space>(CH<space>xxx...)`